gRPC.podspec

```diff
@@ -25,16 +24,13 @@ Pod::Spec.new do |s|
 
   s.subspec 'C-Core' do |cs|
     cs.summary  = 'Core cross-platform gRPC library, written in C.'
-    cs.authors = { 'Craig Tiller'   => 'ctiller@google.com',
-                   'David Klempner' => 'klempner@google.com',
-                   'Nicolas Noble'  => 'nnoble@google.com',
-                   'Vijay Pai'      => 'vpai@google.com',
-                   'Yang Gao'       => 'yangg@google.com' }
 
     cs.source_files = 'src/core/**/*.{h,c}', 'include/grpc/*.h', 'include/grpc/**/*.h'
     cs.private_header_files = 'src/core/**/*.h'
     cs.header_mappings_dir = '.'
-    cs.xcconfig = { 'HEADER_SEARCH_PATHS' => '"$(PODS_ROOT)/Headers/Build/gRPC" "$(PODS_ROOT)/Headers/Build/gRPC/include"' }
+    cs.xcconfig = { 'HEADER_SEARCH_PATHS' => '"$(PODS_ROOT)/Headers/Build/gRPC" '
+                                             '"$(PODS_ROOT)/Headers/Build/gRPC/include"' }
+    cs.compiler_flags = '-GCC_WARN_INHIBIT_ALL_WARNINGS', '-w'
```

Comment:
For now I think what you have is probably better than adding it only for submission. But we should have issues for fixing those warnings.